libtenzir/include/tenzir/multi_series_builder.hpp

```diff
@@ -68,11 +68,12 @@ class record_generator {
 
   /// @brief Creates an explicitly unflattend field. This function does
   /// not respect the builders unflatten setting.
-  auto unflattened_field(std::string_view key,
-                         std::string_view unflatten) -> object_generator;
+  auto unflattened_field(std::string_view key, std::string_view unflatten,
+                         bool* did_unflattend = nullptr) -> object_generator;
```

Comment:
We should probably remove this, depending on the outcome of the conversation above where we use this.